README.md

```diff
@@ -282,7 +282,8 @@ And finally, you are ready to go! Or at least you should be...
 **Iris Mestres** | [Contact me][deliaBlue]
 
 ### Junior Contributors
-
+**Stephanie Padilla** | [MStephPad] [MStephPad] 
```

Comment:
Great job!! Just to finish this part, you must place your phrase behind the link done (also put them in contact otherwise it won't work).